umbral/curvebn.py

```diff
@@ -13,6 +13,8 @@ class CurveBN(object):
     """
     Represents an OpenSSL Bignum modulo the order of a curve. Some of these
     operations will only work with prime numbers
+    By default, the underlying OpenSSL BIGNUM has BN_FLG_CONSTTIME set for
```

Comment:
I'm not sure this new section of docstring adds anything particularly useful, especially repeated four times.